core/src/main/java/org/frankframework/parameters/AbstractParameter.java

```diff
@@ -527,13 +527,21 @@ private String formatPattern(ParameterValueList alreadyResolvedParameters, PipeL
 
 			// get value
 			Object substitutionValue = getValueForFormatting(alreadyResolvedParameters, session, substitutionPattern);
```

Comment:
Ik zou de method `getValueForFormatting()` kunnen aanpassen om een string terug te geven zodat we de complexiteit van de if/else en try/catch niet hier hebben maar dat maakt de method minder generiek dus ik heb gekozen voor de meest recht-toe-recht-aan oplossing.